modules/wpml/wpml-config.php

```diff
@@ -144,19 +146,14 @@ public function init() {
 			foreach ( $keys as $key ) {
 				$name = $this->get_field_attribute( $key, 'name' );
 
-				if ( false === strpos( $name, '*' ) ) {
+				if ( ! $matcher->is_format( $name ) ) {
```

Comment:
I know you kept the same behavior as before. But I'm not very fond of this pattern.
IMHO this class shouldn't be aware if an option has format or not and only pass options and matchers to `PLL_Format_Util()` and get the result.
What do you think to move `PLL_Format_Util::filter_list()` inside `register_or_translate_option()` for instance?

Reply:
I don't know how to do another way, because if it's not a format we don't use `wp_load_alloptions()`.
Also, you say that this class shouldn't be aware of that but you suggest to move this to another method of the same class 😕

Reply:
Resolved after a chat.
There is still a performance issue with `wp_load_alloptions()` IMHO, but since it's the current behavior we may not change it.